Content.Server/GameObjects/EntitySystems/Click/InteractionSystem.cs

```diff
@@ -409,6 +410,8 @@ private void UserInteraction(IEntity player, GridCoordinates coordinates, Entity
                 return;
             }
 
+            playerTransform.LocalRotation = new Angle(coordinates.ToMapPos(_mapManager) - playerTransform.MapPosition.Position);
```

Comment:
Probably best to directly round this to the nearest cardinal direction for now?

While right now the sprite system will automatically round it, that won't forever be the case, and there might be other funky things happening due to this in the future (bounding boxes rotating).

Reply:
Movement code sets this 8-directional, do you want that or 4-directional rounding

Reply:
Oh, hrm. in that case it's fine I guess.

Reply:
I agree that it would make sense to return to later